src/python/WMComponent/RucioInjector/Database/MySQL/GetUnsubscribedDatasets.py

```diff
@@ -49,7 +47,7 @@ def formatToPhEDEx(self, result):
             else:
                 entry['request_only'] = 'y'
             del entry['auto_approve']
-            for key in ['custodial', 'move']:
+            for key in ['custodial']:
```

Comment:
this for loop can be removed and the if statements turned to:
```
                if entry['custodial'] == 1:
                    entry['custodial'] = 'y'
                else:
                    entry['custodial'] = 'n'
```

Reply:
Done with https://github.com/dmwm/WMCore/pull/11181/commits/43a40a017a31b041ecb3a267ba519d2f0d6929f9